### ISSUE TITLE
Error parsing stylesheets that contain a comment at the end

### ISSUE THREAD

**Issue by jakob-metro** [GoogleChromeLabs/container-query-polyfill#8]
I decided to give this a run on a boilerplate WordPress theme we developed. I added the iife script from unpkg.com to the footer of my site as per the instructions.

My stylesheets are compiled via sass and include a sourcemap comment as part of the build process.
```css
/*# sourceMappingURL=style.min.css.map */
```

When this sourcemap comment is included _at the end of the file_ I get an error in the promise.
![image](https://user-images.githubusercontent.com/61976108/143940792-a9c8a993-172b-4121-a438-ace9fde45fa6.png)

It doesn't really matter what the comment at the end is, the script just seems to throw an error if there's _any (valid) css comment_ at the end of the file.

For a reduced test case this file would work without error...
```css
.test {
    background: red;
}
```

...and this file would fail
```css
.test { 
    background: red;
}
/* a comment */
```

Tested in Chrome 96.0.4664.45 (Official Build) (64-bit)

**Comment by jakob-metro** [comment]
Reviewing the readme, it seems that comments interspersed within the stylesheet is discouraged
> Don’t do weird interspersed comments, okay? Like @container /* here’s a comment! */ (min-width: 1px) { ... }. Just don’t.

Nevertheless, the practice of linking sourcemaps is very common practice, so the script should be equipped to handle that case.

**Comment by surma** [comment]
A comment at the end of a file should be fine. This is probably a bug that I can fix.

**Comment by surma** [comment]
Fixed in 3bb4b69
